Update example app

jojo-stripe opened the pull request:
Noticed the example rollup app was out of date, updated it and the README instructions to properly reflect how we test now that the package is publish and we have GA

jorgea-stripe commented on examples/rollup/package.json:
```diff
@@ -4,7 +4,7 @@
   "main": "index.js",
   "dependencies": {
-    "@stripe/connect-js": "file:stripe-connect-js-1.0.0.tgz"
+    "@stripe/connect-js": ">=2.0.1"
```
Shouldn't this be targeting the local via yalc?

jojo-stripe replied:
This was originally targeting local because the package wasn't published yet, but now that it is we should use the published version. This example app is to allow users to run this library from the same repo. Stripe-JS also follows the same example-https://github.com/stripe/stripe-js/blob/master/examples/rollup/package.json

jorgea-stripe replied:
:portalto: https://github.com/stripe/connect-js/pull/54/files#r1355777046

jorgea-stripe commented on examples/rollup/package.json:
```diff
@@ -20,7 +20,6 @@
   "scripts": {
     "build": "rollup -c",
     "server": "FLASK_APP=server.py python -m flask run --port 4242",
-    "local-install": "yalc add @stripe/connect-js@1.0.0 && yalc add @stripe/react-connect-js@1.0.0",
```
How will we test the local changes with this sample?

jojo-stripe replied:
I think the goal of this sample is not to test local changes but to show users how to use this library.

jorgea-stripe replied:
Looks like even StripeJS doesn't have that: https://github.com/stripe/stripe-js/blob/master/CONTRIBUTING.md. I am peaceful with these changes given that, it'd be great if we could document how to use yalc to test local changes in this example in the readme.md of the examples/rollup folder, or in the CONTRIBUTING.md. I can see us asking someone external "can you test ..." however if there are no instructions, that is hard.

jojo-stripe replied:
Added some instructions in readme!

jorgea-stripe reviewed (changes requested):
A couple of questions!

hliana-stripe commented on examples/rollup/src/index.js:
```diff
@@ -22,17 +22,21 @@ const clientSecret = await fetchClientSecret();
   });
-  const payments = connectInstance.create("payments");
-  document.getElementById("payments").append(payments);
+  const onboarding = connectInstance.create("account-onboarding");
```
fyi I ran into this error when testing locally
```You are trying to create an Account Session for an account that is not supported.```
which I suppose is due to my test account having stripe own onboarding, and this is working as intended

Will this be common for others to run into this issue?

jojo-stripe replied:
This example will be merged into master/GA, where the only component that is in GA will be account onboarding. Currently we only support custom accounts for onboarding, so if users try to use a different account type they will run into that error. It's a good callout though! I will add a comment linking to our docs that they need to test with a US custom account

hliana-stripe commented on examples/rollup/server.py:
```diff
@@ -6,21 +6,22 @@
 
 stripe.api_key = 'sk_test_sample_key'
 
-stripe.api_version = '2022-08-01; embedded_connect_beta=v1'
```
did we only want to use the GA api in the sample?

jojo-stripe replied:
Yup! For the beta branch, I will make similar changes but show an example with the components we have in beta instead

hliana-stripe reviewed (commented):
Ran this locally and the install worked! Had a few minor suggestions.
Also, I did get an error that I put in the comments, but i think it's working as intended.

jojo-stripe commented:
@hliana-stripe  @jorgea-stripe  Addressed your comments and added placeholders that align with README as well as comments about using US custom platform and basic yalc instructions

jorgea-stripe commented on examples/rollup/README.md:
```diff
@@ -21,8 +21,13 @@ pip3 install -r requirements.txt
+
+Note: if you are doing local development, you can use yalc to test local changes
+
+1. yalc add @stripe/connect-js # in this sample
```
See https://github.com/stripe/connect-js/pull/55#discussion_r1357426934, this isn't how I've tested it before, however if you are confident it works then I am peaceful :)

jorgea-stripe commented on examples/rollup/server.py:
```diff
@@ -15,6 +15,8 @@
     try:
         account_session = stripe.AccountSession.create(
+          # We currently only support US custom accounts for onboarding
+          # https://stripe.com/docs/connect/get-started-connect-embedded-components#account-onboarding for more info
```
👍

jorgea-stripe commented on examples/rollup/README.md:
```diff
@@ -21,8 +21,13 @@ pip3 install -r requirements.txt
 Run 
-yarn install --> installs all packages
-yarn start --> runs rollup to compile and starts the server
+yarn install # installs all packages
```
Did you also mean to edit the package.json similar to https://github.com/stripe/connect-js/pull/55/files#diff-9722ed0a23a99f56ea234e553da55c4adddbc8c2c3649d4299ecc0643def7648?
